lib/pbench/server/database/models/users.py

```diff
@@ -5,11 +5,24 @@
 from flask_bcrypt import generate_password_hash
 from sqlalchemy import Column, DateTime, Enum, Integer, LargeBinary, String
 from sqlalchemy.orm import relationship, validates
+from sqlalchemy.orm.exc import NoResultFound
 
 from pbench.server.database.database import Database
 from pbench.server.database.models.active_tokens import ActiveTokens  # noqa[F401]
```

Comment:
Note that, with any luck, this import of ActiveTokens (and the apparently unavoidable LGTM warning) shouldn't be necessary since the new `init_db` has all the models imported before it starts. (I hope.) Then again, don't mess with anything else until we get the user lookup problem resolved. :-(

Reply:
Are we ready to mess with this now?